lib/plugins/openaps.js

```diff
@@ -491,6 +491,30 @@ function init(ctx) {
       , intentHandler: alexaLastLoopHandler
     }]
   };
+	
+ function googleHomeForecastHandler(response, next, sbx) {
+    if (sbx.properties.openaps && sbx.properties.openaps.lastEventualBG) {
+      var response = 'The Open APS eventual BG is ' + sbx.properties.openaps.lastEventualBG;
+      next(response);
+    }
+  }
+
+  function googleHomeLastLoopHandler (response, next, sbx) {
+    var response = 'The last successful loop was ' + moment(sbx.properties.openaps.lastLoopMoment).from(moment(sbx.time));
+    next(response);
+  }
+
+  openaps.googleHome = {
+    intentHandlers: [{
+      intent: 'CurrentMetric'
+      , routableSlot: 'metric'
+      , slots: ['openaps', 'openaps forecast', 'forecast']
+      , intentHandler: googleHomeForecastHandler
+    }, {
+      intent: 'LastLoop'
+      , intentHandler: googleHomeLastLoopHandler
+    }]
+  };
 
```

Comment:
And same here - Home and Alexa code for creating the speech user interface should be in the plugin itself. What'd be acceptable for this file is a generic API method that returns a string for speech, which is reused in both Home and Alexa speech implementations, but custom method for both adds a lot of maintenance burden